RealmSwift/BSON.swift

```diff
@@ -18,9 +18,13 @@
 
 import Realm
 
+/// Protocol representing a PartitionValue for synced Realms.
```

Comment:
```suggestion
/// A tag protocol which marks types that can be used as the partition value
/// for synchronized Realms.
```

I'm confused about how this could be used. AFAICT it's only used by the environment key, and I don't see how you can actually open a Realm from the value read from the environment key? If there's somewhere testing the env key I couldn't find it.

Reply:
There are some UI tests testing this, You can see it implemented on the view https://github.com/realm/realm-cocoa/blob/496105721836de76f5a8a37614d75e383f10cf3d/Realm/Tests/SwiftUISyncTestHost/ContentView.swift#L69 and the test https://github.com/realm/realm-cocoa/blob/496105721836de76f5a8a37614d75e383f10cf3d/Realm/Tests/SwiftUISyncTestHostUITests/SwiftUISyncTestHostUITests.swift#L92 for `AsyncOpen`. You can find implementation and tests for `AutoOpen` in the same files